### PR TITLE
add _posts/2022-06-29-#25077.md

### PR DESCRIPTION
**Pull request by MarcoFalke**

### CODE REVIEW

File: _posts/2022-06-29-#25077.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+layout: pr
+date: 2022-06-29
+title: "Fix chain tip data race and corrupt rest response"
+pr: 25077
+authors: [MarcoFalke]
+components: ["Validation"]
```
Comment (MarcoFalke) [inline _posts/2022-06-29-#25077.md:7]:
```suggestion
components: ["validation"]
```

**Comment by glozow** [comment]
@MarcoFalke did you mean to do June 29th or July 27th?

**Comment by MarcoFalke** [comment]
Oh sorry, I misread. I am not sure if I have internet access on July 27th, so feel free to close.

**Review by glozow — COMMENTED** [review]
Looks good, thanks @MarcoFalke! Will merge after date change to 2022-07-06 (re: offline convo). Thanks for the flexibility :pray:

**Comment by MarcoFalke** [comment]
rebased

**Comment by glozow** [comment]
Thanks @MarcoFalke!

**Pull request merged**